server/app.js

```diff
@@ -30,4 +32,12 @@ app.use((err, req, res) => {
   });
 });
 
+// initialize database
+fs.readFile('seed/quotes.json', (err, data) => {
```

Comment:
lets move it to seed directory and call the file `seeder.js`
also let's add a `npm` script to seed the database `npm run seed`

make sure you clear the database before starting to insert

Reply:
@skyshader What about I create a function in `seeder.js` and call that from `server.js` or `app.js`?

Reply:
Nope, app should be independent of seeder. A seeder should be involved independently from the app.js.